invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesList.js

```diff
@@ -0,0 +1,116 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// InvenioRDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React, { Component } from "react";
+// import PropTypes from "prop-types";
+import axios from "axios";
+import { withCancel } from "react-invenio-forms";
+import { Container, Item, Button, Placeholder, Message } from "semantic-ui-react";
+import { Image } from "react-invenio-forms";
+import { CommunitiesListModal } from "./CommunitiesListModal/CommunitiesListModal";
+
+export class CommunitiesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+      communities: null,
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    this.getCommunities();
+  }
+
+  componentWillUnmount() {
+    this.cancellableFetchCommunities?.cancel();
+  }
+
+  getCommunities = async () => {
+    this.cancellableFetchCommunities = withCancel(this.fetchRecordCommunities());
+    this.setState({
+      loading: true,
+      error: null,
+    });
+    try {
+      const res = await this.cancellableFetchCommunities.promise;
+      this.setState({
+        communities: res.data?.hits?.hits,
+        loading: false,
+      });
+    } catch (error) {
+      if (error !== "UNMOUNTED") {
+        this.setState({
+          loading: false,
+          error: i18next.t("An error occurred while fetching communities."),
+        });
+      }
+    }
+  };
+
+  fetchRecordCommunities = async () => {
+    // TODO: Update endpoint to be record specific
+    return await axios.get("/api/communities", {
+      headers: {
+        Accept: "application/vnd.inveniordm.v1+json",
+      },
+    });
+  };
+
+  render() {
+    const { communities, loading, error } = this.state;
+    const communityItems = communities?.map((community, index) => {
+      if (index > 2) return null;
```

Comment:
if you consider the query from the previous comment, this line could be probably removed

Reply:
Yes